tests/integration/auth/keeper/account_retriever_test.go

```
@@ -1,45 +1,84 @@
+	"google.golang.org/grpc/credentials/insecure"
 
-	authTest "github.com/cosmos/cosmos-sdk/tests/integration/auth/keeper"
-	"github.com/cosmos/cosmos-sdk/testutil/network"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	srvconfig "github.com/cosmos/cosmos-sdk/server/config"
+	servergrpc "github.com/cosmos/cosmos-sdk/server/grpc"
+	simtestutil "github.com/cosmos/cosmos-sdk/testutil/sims"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/keeper"
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
```

golangci-lint (1.62.2)

ST1019: package "github.com/cosmos/cosmos-sdk/x/auth/types" is being imported more than once

(stylecheck)